dev/reference/transactions/transaction-pessimistic.md

```diff
@@ -30,56 +26,37 @@ Pessimistic transactions in TiDB behave similarly to those in MySQL. See the min
 
 - TiDB supports both the optimistic transaction mode and pessimistic transaction mode in the same cluster. You can specify either mode for transaction execution.
 
-## Methods to enable the pessimistic transaction mode
-
-The pessimistic transaction mode is disabled by default because it is currently an experimental feature. Before enabling it, you need to add the following setting in the configuration file:
-
-```
-[pessimistic-txn]
-enable = true
-```
+## Usage of pessimistic transaction mode
 
-When `enable` is set to `true`, the default transaction mode in TiDB is still optimistic. To enable the pessimistic transaction mode, choose any of the following methods that suits your needs:
+To enable the pessimistic transaction mode, choose any of the following three methods that suits your needs:
 
 - Use the `BEGIN PESSIMISTIC;` statement to start the transaction in the pessimistic transaction mode. You can write it in comment style as `BEGIN /*!90000 PESSIMISTIC */;` to make it compatible with the MySQL syntax.
 
 - Execute the `set @@tidb_txn_mode = 'pessimistic';` statement to allow all the explicit transactions (namely non-autocommit transactions) processed in this session to be in the pessimistic transaction mode.
 
-- Enable the pessimistic transaction mode in the configuration file. This allows all transactions (except auto-committed single-statement ones) to adopt the pessimistic transaction mode.
+- Execute the `set @@global.tidb_txn_mode = 'pessimistic';` statement to allow all newly created sessions of the entire cluster to adopt the pessimistic transaction mode and execute the explicit transactions.
```

Comment:
```suggestion
- Execute the `set @@global.tidb_txn_mode = 'pessimistic';` statement to allow all newly created sessions of the entire cluster to adopt the pessimistic transaction mode and execute explicit transactions.
```

Reply:
Should be `to` or `when` instead of `and` in `and execute the explicit transactions`

Reply:
`apply` is better than `adopt`.